lib/modules/contributors/services/github_service.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+import 'dart:convert' as convert;
+import 'package:http/http.dart' as http;
+import '../models/contributor.dart';
+import '../models/user.dart';
+import 'open_contributors_cache_manager.dart';
+
+
+class InvalidResponseError extends StateError {
+  InvalidResponseError(String msg) : super(msg);
+}
+
+class GithubService {
+  static const BASE_URL = 'https://api.github.com';
+  static const CREDENTIALS = '?client_id=2a73a1450d7b195e0d62&client_secret=29e82f1e6ad8a081d47fcf0a338b772b98166388';
```

Comment:
are you sure you want those here?

Reply:
For now yes ), later i will make a package out of it and require it as a param.

Reply:
![tumblr_nbv8jkKpnJ1tc258so2_r1_1280](https://user-images.githubusercontent.com/51994215/73023670-b4288f80-3e34-11ea-8598-fd93f673970d.png)